src/super_gradients/common/deprecate.py

```diff
@@ -112,7 +112,7 @@ def override(self, **entries):
 
     def decorator(func):
         def wrapper(*args, **training_params):
-            if deprecated_tparam_name in training_params:
+            if training_params.get(deprecated_tparam_name):
```

Comment:
Should we also check whether result of get is empty dict (`{}`)? or this should not happen?

Reply:
Empty dict can happen but it will also be ignored (`{}` is seen as `False` in `if` statements).
This is precisely what we want to cover. 
if you have `training_params = {..., "loss": {"MyLoss": {"param": "value}}, "criterion_params": {}}` Then `criterion_params` is meaningless.